### ISSUE TITLE
[Accessibility]: Search match highlight background color has insufficient text contrast

### ISSUE THREAD

**Issue by limitedmage** [sourcegraph/sourcegraph#41378]
### Audit type

ARC Toolkit

### User journey audit issue

https://github.com/sourcegraph/sourcegraph/issues/34190

### Problem description

Search match highlight background color has insufficient text contrast in both light and dark modes.

![image](https://user-images.githubusercontent.com/206864/188705296-369cd3aa-a0fd-4451-a301-331d8df0b5d3.png)

![image](https://user-images.githubusercontent.com/206864/188705311-02456ec0-7aba-4a74-b9e6-eeada09d58ba.png)

### Expected behavior

The highlight color has sufficient contrast

### Additional details

Related issues:
- https://github.com/sourcegraph/sourcegraph/pull/41296
- https://github.com/sourcegraph/sourcegraph/pull/39845

### Assigning labels

- [X] Please give this issue an estimate by applying a label like `estimate/Xd`, where X is the estimated number of days it will take to complete.
- [X] If this issue is specific to a specific Sourcegraph product, please assign the appropriate team label to this issue.
- [X] If this issue will require input from designers in order to complete, please assign the label `needs-design`.
- [ ] If you are confident that this issue should be fixed by GitStart, please assign the label `gitstart`.

### Owner

This issue will be fixed by my team, I have assigned a relevant member to it, or I will do so in the near future.

/cc @benvenker @lguychard

**Comment by limitedmage** [comment]
@almeidapaulooliveira assigning this to you to pick a color. Assign back to me when a color has been selected.

**Comment by almeidapaulooliveira** [comment]
@limitedmage, I explored the colors only for the **path match** on this [Figma file. ](https://www.figma.com/file/DmIlwMDNaJ5TBa6FYnR6WK/Search-match-highlight-background-color-has-insufficient-text-contrast?node-id=2%3A24)

**Comment by limitedmage** [comment]
#41537 fixes this issue, but only for search result headers, not for content. We have decided not to address content for the time as it's very complex to figure out with all the syntax highlighting themes.
